src/http-auth-interceptor.js

```diff
@@ -5,6 +5,12 @@
  * (c) 2012 Witold Szczerba
  * License: MIT
  */
+
+/* commonjs package manager support (eg componentjs) */
+if (typeof module !== "undefined" && typeof exports !== "undefined" && module.exports === exports){
+  module.exports = 'http-auth-interceptor';
```

Comment:
It's because angular already has it's own module system which is independent from ES6/CommonJS. So with angular modules one would usually only be interested in their names. See [angular-material](https://github.com/angular/bower-material/blob/master/index.js) for example.